index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Aamir Asaram</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet"
+  href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <header>
+    <nav class="header">
+      <img src="assets/images/menu.png" class="menu-btn"">
+    </nav>
+  </header>
+  <div class="bg">
+    <div class="intro">
+      <div>
+        <h2 class="blue heading">Hey there. I'm Aamir.</h2>
+        <h1 class="green heading">I'm a software developer</h1>
+      </div>
+      <article>
+        <p class="blue">I can help you build a product, feature or website.
+        Look through some of my work and experience! If you like
+        what you see and have a project you need coded, don't hesitate
+        to contact me.</p>
+      </article>
+
+      <div class="social">
+        <ul>
+          <!-- </a> -->
+          <a href="https://twitter.com/aaaaamir_"><li><button class="social-btn"><i class="fa fa-twitter"></i></button></li></a>
+          <a href="https://www.linkedin.com/in/aamir-asaram-089802213/"><li><button class="social-btn"><i class="fa fa-linkedin"></i></button></li></a>
+          <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ"><li><button class="social-btn"><i class="fa fa-medium"></i></button></li></a>
+          <a href="https://github.com/aamir-asaram"><li><button class="social-btn"><i class="fa fa-github"></i></button></li></a>
+          <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ"><li><button class="social-btn"><i class="fa fa-angellist"></i></button></li></a>
+        </ul>
+      </div>
+      <div class="down"></div>
+      </div>
+    </div>
+  </div>
```

Comment:
- [ ] As mentioned by the previous reviewer and for the project requirements, kindly ensure that your design sticks as much as possible to your chosen Figma template. You should use the same `fonts(size, weight,...)` on different text, correct `spacing` between and around different elements, `colors`, `size`, `line height`,.... Make sure you check all information from the Figma template and apply them on your project.

| Figma design | Your Design |
| ------------------- | ------------------ |
| ![Screenshot from 2023-04-27 07-56-36](https://user-images.githubusercontent.com/83042742/234772822-3fe92fbb-7075-4f65-bd8b-1467315fdabd.png) | ![Screenshot from 2023-04-27 07-57-43](https://user-images.githubusercontent.com/83042742/234772626-3266e53b-7c57-4417-956f-5c2ba352d553.png) |

Reply:
Hi @tresorsawasawa, I'm struggling to get the spacing of the icons and the sides of the page to match. Have you got any tips?